packages/main/src/Avatar.js

```diff
@@ -208,6 +208,35 @@ const metadata = {
 			propertyName: "image",
 			type: HTMLElement,
 		},
+		/**
+		 * Defines the optional <code>ui5-badge</code> that will be used for visual affordance.
+		 * Use <code>slot="badge"</code> together with the <code>avatar-badge</code> attribute for proper styling.
```

Comment:
I think it's better to strip the part  **"together with the <code>avatar-badge</code> attribute for proper styling."**. 
If app developers use "ui5-badge" inside the Avatar's "badge" slot, no additional effort is needed right?

Reply:
There's no way to scope the styles without an attribute, otherwise they will apply to all slotted badges, e.g. ones used for a button, instead of avatar.

Reply:
Done